Subtester/Assets/Scripts/PurchasesListener.cs

```diff
@@ -769,6 +773,199 @@ void ShowInAppMessages()
         Purchases.InAppMessageType.PriceIncreaseConsent, Purchases.InAppMessageType.Generic, Purchases.InAppMessageType.WinBackOffer });
     }
 
+    void PurchaseProductForWinBackTesting()

```

Comment:
Hmm what's the difference between this one and the existing purchase methods? Same for `PurchasePackageForWinBackTesting`

Reply:
The only difference is that it specifically purchases the product that has a win-back offer associated with it in the SKConfig file! If you purchase a different product, it won't have a win-back offer on it and you won't be able to test the functionality.

Reply:
Right... then I would say we should probably do a small refactor so only the product id becomes a parameter

Reply:
Did a small refactor so that these functions just call the existing purchase functions under the hood in  485633996fa3daf29ffd04344108c840ca8e0ed2 😄